src/main/resources/swagger/api-docs.yaml

```diff
@@ -487,6 +487,11 @@ definitions:
       labels:
         type: object
         description: The labels to be placed on the cluster. Of type Map[String,String]
+      errors:
```

Comment:
Should notify Bru (can't find his github handle) that the cluster swagger model is changing.